src/main/java/mate/academy/dao/UserDao.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.User;
+
+public interface UserDao {
+    User add(User user);
+
+    Optional<User> get(Long id);
```

Comment:
You don't use  `Optional<User> get(Long id); and List<User> getAll();` methods, remove them